src/NosCore.GameObject/ComponentEntities/Interfaces/ICharacterEntity.cs

```diff
@@ -46,6 +46,9 @@ public interface ICharacterEntity : INamedEntity, IExperiencedEntity
 
         int DignityIcon { get; }
 
+        bool Invisible { get; }
+
+        bool InvisibleGm { get; }
```

Comment:
Im quite sure InvisibleGm can be replace by something more explicit like InvisibleOnList